GuardiansoftheSpectrum/app/src/main/java/edu/colorado/gots/guardiansofthespectrum/BaseActivity.java

```diff
@@ -55,6 +57,10 @@ public boolean onCreateOptionsMenu(Menu menu) {
         MenuInflater mMenuInflater = getMenuInflater();
         mMenuInflater.inflate(R.menu.our_menu, menu);
 
+        Toolbar mToolbar = (Toolbar) findViewById(R.id.toolbar);
```

Comment:
I am commenting them out for the commit, but without these lines the toolbar does not show options on the main menu. I remember seeing that loop on your device, don't see it in debug on Galaxy. We can look at it tomorrow, if we have time.